build.js

```diff
@@ -11,6 +11,7 @@ const mappers = { // https://www.nesdev.org/wiki/Mapper
     3: 'CNROM',
     4: 'MMC3',
     5: 'MMC5',
+    255: 'Autodetect MMC1/CNROM',
```

Comment:
255 is https://www.nesdev.org/wiki/INES_Mapper_255

we could just change the key here to `auto` I think

Reply:
I like that, auto would be more clear.  I picked a number to keep the types the same but I don't know that they have to be.  I chose 255 because it's a "skull" mapper so would never be used.  If cc65 doesn't care about string/number mixing then I'll change to auto.

Reply:
It didn't like "auto".  There may still be a way to use a string but for now it looks like it needs to be a number.

Reply:
changed to 1000